app/mailers/user_mailer.rb

```diff
@@ -9,12 +14,11 @@ def notify_subscriber(group, user, message)
     mail(to: @user.email, subject: 'Epic Logger Subscriber notification')
   end
 
-  def member_invitation(website_id, email, website_member_id, inviter_id)
-    @user = User.find(inviter_id)
-    @website = Website.find(website_id)
-    @token = WebsiteMember.find(website_member_id).invitation_token
-    @email = email
-    mail(to: @email, subject: 'Epic Logger Invite Users')
+  def member_invitation(invite_id)
```

Comment:
considering what i wrote above please send the full invite record here and not the id. If you are using deliver_later and action mailer.